src/App.js

```diff
@@ -1,40 +1,9 @@
+import Calculator from './components/Calculator';
+
 function App() {
   return (
     <div className="App">
-      <div className="calculator">
-        <div className="display">
-          0
-        </div>
-        <div className="buttons line1">
-          <button type="button">AC</button>
-          <button type="button">+/-</button>
-          <button type="button">%</button>
-          <button type="button">/</button>
-        </div>
-        <div className="buttons line2">
-          <button type="button">7</button>
-          <button type="button">8</button>
-          <button type="button">9</button>
-          <button type="button">x</button>
-        </div>
-        <div className="buttons line3">
-          <button type="button">4</button>
-          <button type="button">5</button>
-          <button type="button">6</button>
-          <button type="button">-</button>
-        </div>
-        <div className="buttons line4">
-          <button type="button">1</button>
-          <button type="button">2</button>
-          <button type="button">3</button>
-          <button type="button">+</button>
-        </div>
-        <div className="buttons line5">
-          <button type="button">0</button>
-          <button type="button">.</button>
-          <button type="button">=</button>
-        </div>
-      </div>
+      <Calculator />
```

Comment:
- [ ] Good job so far 👍 As per the requirements, all your components should be class-based. Kindly convert this `app` component from a function to a class 😄